### DISCUSSION TITLE
Is this possible to have additional partition.

### DISCUSSION THREAD

**Discussion by nkpro2000** [ventoy/Ventoy#778] [Q&A] [answered]
Is possible to have one or more additional partition and to make ventoy never search for ISOs in these partition.

> It is ok, if required to create partitions after the installation of ventoy

**Comment by chromer030** [comment]
After formatting and creating disk by Ventoy you can create more partition via any partition manager you like (for example gparted).

* By default, Ventoy will search all the directories and sub-directories in the 1st partition to find all the ISO/IMG... files.
You can change the default search root in Global Control Plugin with a VTOY_DEFAULT_SEARCH_ROOT option, then Ventoy will only search the specified directory and its sub-directories.
Besides, you can put a .ventoyignore file under some directory, Ventoy will skip this directory and its sub-directories when searching files.
Finally, Ventoy will sort all the files found by their names and show the boot menu.

For more configuration and options : https://www.ventoy.net/en/plugin_imagelist.html

**Comment by nkpro2000** [comment, marked as answer]
### Observation:
#### Working properly:
`[ VENTOY PARTITION ] [ VTOYEFI PARTITION ] [ Additional Partitions ]`
#### Not Working:
`[ VENTOY PARTITION ] [ Additional Partitions ] [ VTOYEFI PARTITION ]`
`[ VENTOY PARTITION ] [ Additional Partitions ] [ VTOYEFI PARTITION ] [ Additional Partitions ]`

### Result:
VENTOY Partition (where Ventoy search for ISOs), VTOYEFI Partition (32 MB partition created on installation) must be 1st and 2nd partitions. And we can add any number of additional partitions following the above partitions.

> After formatting and creating disk by Ventoy you can create more partition via any partition manager you like (for example gparted).
>> as said by @chromer030

**Reply by ventoy** [reply]
Yes, it was noted here:
https://www.ventoy.net/en/doc_disk_layout.html#reserve_space

**Reply by nkpro2000** [reply]
my bad, i didn't notice that
